index.js

```diff
@@ -4,7 +4,26 @@
 import element from 'magic-virtual-element';
 
 export default {
-  render: function () {
-    return (<div></div>);
+  render: function ({ props }) {
+    return (<div class='news-ticker'>
+      <span class='news-ticker__label'>{ props.label || 'The Latest News'}</span>
+      <span class='news-ticker__slider'>
+        {props.children}
+      </span>
+    </div>);
+  },
+  afterMount: function (component, el) {
```

Comment:
I think that we should save the `setInterval` to the state (using `setState`) and clear it in `beforeUnmount` so that the interval is removed when the element is unmounted. Right?